CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 2.15 (June 2020)
+- Login rearchitecture
+- Support for OpenId Connect
+- Biometrical lock
```

Comment:
I'd specify the new implementation, so this is not a new feature. Something like `Biometrical lock using the Android 10 library` or similar.